packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -134,7 +134,7 @@ describe('--schema from parent directory', () => {
 function cleanSnapshot(str: string): string {
   return str
     .replace(/\d+ms/g, 'XXms')
-    .replace(/\d+s/g, 'XXs')
+    .replace(/\d+s/g, 'XXms')
```

Comment:
I think that we had `4.XXs` because that's what the regex matched, it was not designed to match full decimal numbers.
```suggestion
    .replace(/\d+(?>.\d+)?s/g, 'XXms')
```

Reply:
Indeed, thanks!

Reply:
It's an invalid regex, I fixed it in a new commit